modules/sspBCBidAdapter.js

```diff
@@ -199,6 +199,22 @@ const applyClientHints = ortbRequest => {
   ortbRequest.user = { ...ortbRequest.user, ...ch };
 };
 
+const applyTopics = (validBidRequest, ortbRequest) => {
+  const userData = validBidRequest.ortb2?.user?.data || [];
+  const topicsData = userData.filter(dataObj => {
```

Comment:
the `ortb2` object you get as input is meant to be usable as-is for your ORTB exchange, so I'm always curious to learn why the extra processing. 

From what I see here you prefer to:

 - *not* get all the available user data segments, but only topics;
 - *not* get information about who provided the topics (`data[].name`).

Both choices seem odd to me, which is why I note them here, but of course you're free to make them.

Reply:
Our exchange is, unfortunately, not fully ortb-compliant  - that's why we:
a) trim data in topics
b) custom-build other user segments (see applyClientHints etc)

The next major version of the adapter should use ortbConverter, but we're waiting for the changes on the backend (to eliminate the need for overrides on the adapter side).